package.json

```diff
@@ -44,6 +44,8 @@
     "@openedx/paragon": "^22.16.0",
     "@redux-beacon/segment": "^1.0.0",
     "@reduxjs/toolkit": "^1.5.1",
+    "@testing-library/jest-dom": "^6.6.3",
+    "@testing-library/react": "^16.2.0",
```

Comment:
These should stay dev dependencies. Is there a specific reason you moved them to be direct dependencies?